language/move-lang/src/parser/mod.rs

```diff
@@ -34,12 +35,12 @@ pub(crate) fn parse_program(
 )> {
     let targets = find_move_filenames(targets, true)?
         .iter()
-        .map(|s| leak_str(s))
-        .collect::<Vec<&'static str>>();
+        .map(|s| Symbol::from(s.as_str()))
```

Comment:
Yeah... let's add `From<String>` for `Symbol` before landing this. Finding all of these spots is going to be painful